dao/postgres/schemas/core.sql

```diff
@@ -130,6 +130,7 @@ CREATE TABLE wicked.subscriptions (
     plan_id character varying(128) NOT NULL,
     api_id character varying(128) NOT NULL,
     client_id character varying(256),
+    api_group character varying(128),
```

Comment:
Philosophical question whether this should be in here or not...

Reply:
This is needed for the tests to work. Tests build the proper schema from this schema file. I didn't had it before.

Reply:
See my new comment - you should be able to take this out if you run the "populate" step at the right point in time - after you have added the field. This should also work for the tests, they also run through the update process.

Reply:
done